spring-cloud-starter-stream-source-sftp-dataflow/README.adoc

```diff
@@ -0,0 +1,130 @@
+//tag::ref-doc[]
+= SFTP Data Flow Source
+
+This is a specialized version of the `sftp` source which always outputs a Data Flow Task Launch Request as JSON,
+compatible with the https://github.com/spring-cloud-stream-app-starters/tasklauncher-data-flow[tasklauncher-data-flow
+] sink.
+
+NOTE: A task launch request posted to the Data Flow Server API is much simpler to use than the `TaskLaunchRequest`
+supported by the original `sftp` source, designed to launch tasks using one of the provided platform specific task
+launchers. Using a platform specific task launcher makes it possible to launch tasks when a Data Flow server is not
+deployed, but requires several additional configuration parameters.
+
+This source transfers files using the SFTP protocol from the `remote` directory to the `local` directory where the application is deployed.
+
+NOTE: When running on a cloud platform such as (Cloud Foundry 2.3+ or Kubernetes), it is recommended that the local
+directory
+reference a shared mount path provided by platform volume services which can also be accessed from the launched task
```

Comment:
directory reference**s** ?

Reply:
`reference` is correct here https://en.wikipedia.org/wiki/English_subjunctive

Reply:
OK! The `is` verb is missed in between `reference` and `a shared` 😄